portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/Minified.testcase

```diff
@@ -45,7 +45,7 @@ definition {
 
 	@description = "Check for errors in the browser console on the jQuery page"
 	@priority = 3
-	test VerifyErroWhenJQueryEnabled {
+	test VerifyErrorWhenJQueryEnabled {
 		property test.name.skip.portal.instance = "Minified#VerifyErroWhenJQueryEnabled";
```

Comment:
```suggestion
		property test.name.skip.portal.instance = "Minified#VerifyErrorWhenJQueryEnabled";
```

Reply:
Weired, I could swear I had changed this line... Deja vu 👻